lib/qtermwidget.cpp

```diff
@@ -698,6 +698,20 @@ void QTermWidget::setBlinkingCursor(bool blink)
     m_impl->m_terminalDisplay->setBlinkingCursor(blink);
 }
 
+void QTermWidget::setBidiEnabled(bool enabled)
+{
+    if (!m_impl->m_terminalDisplay)
+        return;
+    m_impl->m_terminalDisplay->setBidiEnabled(enabled);
+}
+
+bool QTermWidget::isBidiEnabled()
+{
+    if (!m_impl->m_terminalDisplay)
+        return;
```

Comment:
I'm not sure if this will ever happen, but it should return something.

Reply:
:D So its better to be removed.

Reply:
I haven't actually tested it. If things are still fine after dropping the check, feel free to go ahead :)

I opened #156 as a reminder for similar patterns.